tools/expotools/src/commands/RemoveSDKVersion.ts

```diff
@@ -0,0 +1,47 @@
+import chalk from 'chalk';
+import { Command } from '@expo/commander/typings';
+
+import * as IosVersioning from '../versioning/ios';
+import { getExpoRepositoryRootDir } from '../Directories';
+import { getOldestSDKVersionAsync } from '../ProjectVersions';
+
+interface ActionOptions {
+  platform: string;
+  sdkVersion?: string;
+}
+
+const EXPO_DIR = getExpoRepositoryRootDir();
+
+async function action(options: ActionOptions) {
+  if (!options.platform) {
+    throw new Error('Run with `--platform <ios | android>`.');
+  }
+
+  const sdkVersion = options.sdkVersion || await getOldestSDKVersionAsync(options.platform);
```

Comment:
I think making it an explicit argument wouldn't be an inconvenience

Reply:
I do want to make all these commands as easy and as short as possible and also make all of them prepared to be run on the CI, where next/oldest SDK version would need to be calculated somehow. The cost of making it optional was equal to zero, but the profit will be bigger - just trust me 😜

Reply:
I don’t think the CI argument is valid here, I wouldn’t trust the CI to add/remove SDKs any time soon. Also, I would vote for all the methods  to be as idempotent as possible to make any failures easy to fix and debug (not have to clean, checkout, etc.; more of a high level thought). Making this command calculate the oldest ABI by itself may introduce flows like:
- delete ABI
- delete ABI failed
- fixed something jn the script
- run the script again
- *whoops, forgot to checkout changes*

I didn’t request changes on this PR on purpose, so I won’t veto this, but I’m not sure if the convenience/explicity level isn’t exceeded here. 😅

Reply:
Imho this flow is also valid without calculating the oldest ABI, I'd say checkout is obligatory as some commands may fail or do weird things if you didn't remove previously generated files 😛
Maybe, as a compromise, we can add inquirer prompt to confirm the calculated SDK version or allow to put different one? 🤔

Reply:
> Maybe, as a compromise, we can add inquirer prompt to confirm the calculated SDK version 

This sounds good to me. We need to make sure inquirer doesn't block CI accidentally too, so it might be good to either exit with a non-zero code or print a clear message that we are defaulting to SDK XX.0.0.